modules/jaggery-apps/app-store-web/controllers/top-assets.jag

```diff
@@ -26,7 +28,7 @@ require('/modules/store.js').exec(function (ctx) {
             options = {},
             types = store.assetTypes(),
             search = require('/modules/search.js'),
-            utility = require('/modules/util.js');
+            utility = require('/modules/util.js'),isSubscribed;
```

Comment:
Move isSubscribed variable declaration to the next line, for the sake of readability.